prototypes/user-preferences/src/App.js

```diff
@@ -0,0 +1,155 @@
+
+function App() {
+  const [value1, setValue1] = useState(60);
+  const [value2, setValue2] = useState(20);
+  const [value3, setValue3] = useState(1600);
+  const [value4, setValue4] = useState(150);
+  const [value5, setValue5] = useState(180);
+
+  const [selectedRestriction, setSelectedRestriction] = useState(null);
+  const restrictions = [
+    { name: "gluten free", code: "GF" },
+    { name: "dairy free", code: "DF" },
+    { name: "nut free", code: "NF" },
+  ];
+
+  return (
+    <div className="App">
+      <header className="App-header">
+        <Panel header="Macro Goals">
+          <p className="m-0">
+            <label>Protein:</label>{" "}
+            <InputNumber
+              inputId="stacked-buttons"
+              value={value1}
+              onValueChange={(e) => setValue1(e.value)}
+              inputClassName="quantity-input"
+            />{" "}
+            grams
+          </p>
+          <p className="m-0">
+            <label>Carbs:</label>{" "}
+            <InputNumber
+              inputId="stacked-buttons"
+              value={value2}
+              onValueChange={(e) => setValue2(e.value)}
+              inputClassName="quantity-input"
+            />{" "}
+            grams
+          </p>
+          <p className="m-0">
+            <label>Fat:</label>{" "}
+            <InputNumber
+              inputId="stacked-buttons"
+              value={value1}
+              onValueChange={(e) => setValue1(e.value)}
+              inputClassName="quantity-input"
+            />{" "}
+            grams
+          </p>
+          <p className="m-0">
+            <label>Calories:</label>{" "}
+            <InputNumber
+              inputId="stacked-buttons"
+              value={value3}
+              onValueChange={(e) => setValue3(e.value)}
+              inputClassName="quantity-input"
+            />{" "}
+            kcals
+          </p>
+        </Panel>
+        <Panel header="Dietary Restrictions">
+          <p className="m-0">
+            {/* 
+          TO IMPLEMENT IDEAL FUNCTIONALITY FOR DIETARY RESTRICTION CARD
+
+          -You have a list of selected items that render like the orange parts. Each would have an x to remove it from that selected items list.
+          -Next you have a dropdown that contains all the restrictions. When you select one, you add it to the selected items list
+
+          -Create a restrictions array that generates the orange tags for you.
+          -for each item in that array, add a button that removes it from the array
+          -Then add an autocomplete to append stuff to the array whenever an item from the autocomplete is selected 
+          
+          */}
+
+            {/* <AutoComplete
+              field="name"
+              multiple
+              value={selectedRestriction}
+              // suggestions={filteredCountries}
+              // completeMethod={search}
+              onChange={(e) => setSelectedRestriction(e.value)}
+            /> */}
```

Comment:
```suggestion
            <AutoComplete
              field="name"
              suggestions={allDietaryRestrictions}
              completeMethod={search}
              onChange={(e) => addDietaryRestrictions([...dietaryRestrictions, e.value])}
            />
```

Reply:
oh whoops left some residual code from the countries example in the prime react docs